docs/content/Auth/Auth0-Guide.md

```diff
@@ -77,7 +84,7 @@ Cube.js.
 ```javascript
 function (user, context, callback) {
   const namespace = "http://localhost:4000/";
-  context.accessToken[namespace] =
+  context.idToken[namespace] =
```

Comment:
@hassankhan @ovr Just want to make sure it is correct. We're having `idToken` in one example and `accessToken` in different.

Reply:
@keydunov I'd incorrectly used `accessToken` before; it should be `idToken` for both examples. Tested and verified using the OpenID Connect Playground:

![image](https://user-images.githubusercontent.com/1781985/109154749-a4c9f300-7766-11eb-8865-84953269223f.png)

Reply:
We should use `accessToken`, because `idToken` is used for OpenID Connect spec and looks likes it's deprecated and will be removed.

Reply:
I've just tested this using [the Auth0 example](https://github.com/cube-js/cube.js/tree/master/examples/auth0) and the custom claims only seem to work when added to `idToken`, **not** `accessToken`.